chapter_introduction/index_vn.md

```diff
@@ -1785,7 +1789,14 @@ we must account for the way its actions might
 impact the future observations of the agent.
 -->
 
-*dịch đoạn phía trên*
+Sự đơn giản của học ngoại tuyến có sự hấp dẫn riêng của nó.
+Ưu thế là ta chỉ cần quan tâm đến vấn đề nhận dạng khuôn mẫu mà không cần phân tâm tới những vấn đề khác.
+Nhược điểm lại là khả năng cấu thành bài toán khá hạn chế.
```

Comment:
```suggestion
Nhưng nhược điểm là sự hạn chế trong việc định hình các bài toán.
```

Chỗ này mình có lẽ cần thảo luận thêm. Mấy bác tác giả viết hơi cụt cụt, làm mình toàn phải đọc xung quanh rồi mới hiểu được ý.

Reply:
Em thấy anh sửa vậy OK rồi nên commit luôn.